apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/OutpatientClaimTransformerV2Test.java

```diff
@@ -444,13 +444,16 @@ public void shouldHaveDiagnosesMembers() {
     assertTrue(cmp4.equalsDeep(diag4));
 
     DiagnosisComponent diag5 =
-        TransformerTestUtilsV2.findDiagnosisByCode("R1122", eob.getDiagnosis());
+        TransformerTestUtilsV2.findDiagnosisByCode("B20", eob.getDiagnosis());
 
     DiagnosisComponent cmp5 =
         TransformerTestUtilsV2.createDiagnosis(
             // Order doesn't matter
             diag5.getSequence(),
-            new Coding("http://hl7.org/fhir/sid/icd-10", "R1122", null),
+            new Coding(
+                "http://hl7.org/fhir/sid/icd-10",
+                "B20",
+                "HUMAN IMMUNODEFICIENCY VIRUS �HIV� DISEASE"),
```

Comment:
Whats with the weird encoding here?

Reply:
Thats unfortunately what our transformer code does with these brackets that are provided with the display for HIV.